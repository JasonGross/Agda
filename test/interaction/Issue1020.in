top_command (cmd_load currentFile [])
IOTCM currentFile None Indirect (Cmd_give 0 (Range [Interval (Pn Nothing 192 10 10) (Pn Nothing 204 10 22)]) "Σ[ x ∈ ⊤ ] ?")
