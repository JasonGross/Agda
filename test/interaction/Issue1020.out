Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue1020 (Issue1020.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue1020.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Set\n" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> (agda2-give-action 0 'no-paren)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?1 : Set\n" nil)
((last . 1) . (agda2-goals-action '(1)))
Agda2> 